### DISCUSSION TITLE
CSS and Images are not loading in google cache and  website is built using next js 14

### DISCUSSION THREAD

**Discussion by himanshu3738** [vercel/next.js#64240] [App Router]
CSS and Images are not loading on google cache and i can only see texts and icons , 
webapp is built using next js 14 , tailwind css.
i had gone to google cache page using cache:domain_name

![Screenshot 2024-04-08 122830](https://github.com/vercel/next.js/assets/145534135/f4e184c1-11c4-4408-8c37-810d0cd31fce)
![Screenshot 2024-04-08 122748](https://github.com/vercel/next.js/assets/145534135/6f8df725-965e-4620-8df9-c8b20e205675)

**Comment by evansgreatest** [comment]
Use the next/image component for images Instead

**Reply by himanshu3738** [reply]
I am using next/image for the same , but what about css problem
